Source/SwiftLintFramework/Rules/Metrics/LargeTupleRuleExamples.swift

```diff
@@ -0,0 +1,74 @@
+struct LargeTupleRuleExamples {
```

Comment:
pulling out the examples into a separate file like this seemed to a strategy used elsewhere in the codebase to add Swift version-specific examples, as seen in `ImplicitGetterRuleExamples.swift`, but let me know if there some better way to handle this.